python/tests/core/test_statistics_engine.py

```diff
@@ -30,15 +31,16 @@
 
 
 engine._engine_type = "python"
-fg = feature_group.FeatureGroup(
-    name="test",
-    version=1,
-    featurestore_id=99,
-    primary_key=[],
-    partition_key=[],
-    id=10,
-    stream=False,
-)
+with mock.patch("hopsworks_common.client.get_instance"):
```

Comment:
Is this relevant to this PR?

Reply:
Yes, since `FeatureGroup.__init__` calls `VariableApi.__init__`.